topsort-catalog-service.yml

```diff
@@ -23,6 +23,53 @@ servers:
 tags:
   - name: Catalog
     description: Integration for Marketplaces' catalog queries.
+  - name: Models
+    x-displayName: All Models
+    description: |
+      ## GetProductsByID Request
+      <SchemaDefinition schemaRef="#/components/schemas/GetProductsByIDRequest" />
+
+      ## GetProductsByID Response
+      <SchemaDefinition schemaRef="#/components/schemas/GetProductsByIDResponse" />
+
+      ## Paginated Response
+      <SchemaDefinition schemaRef="#/components/schemas/PaginatedResponse" />
+
+      ## Product
+      <SchemaDefinition schemaRef="#/components/schemas/Product" />
+
+      ## Products Response
```

Comment:
Maybe we could sort the models alphabetically.

We may also want to remove de `XXResponse` as one will only be using the `PaginatedXXResponse`.

Reply:
The XXResponse is part of the definition of PaginatedXXResponse, but yes, it might be too much.